### PR TITLE
Add team section

### PR DESCRIPTION
**Pull request by tkyi**
Help:
- not sure what the titles are for people
- should I have links to everyone else's Githubs?
- is anyone missing from the list?

Related to https://github.com/screwdriver-cd/screwdriver/issues/322

screen shot 2016-12-08 at 11 47 11 am

### CODE REVIEW

File: _includes/team.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                            {% endfor %}
+
+                        </ul>
+                    </div>
+                </div>
+                {% endfor %}
+            </div>
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+                    <p class="large text-muted">Our experiences have inspired us to set out to build an intuitive, sophisticated tool to boost developer productivity. The Screwdriver team is based in Sunnyvale, CA, and consists of Dao Lam, Darren Matsumoto, Jeremiah Wuenschel, Min Zhang, Noah Katzman, Peter Peterson, St.John Johnson, and Tiffany Kyi.</p>
```
Comment (stjohnjohnson) [inline _includes/team.html]: Space between `St.` and `John` please.

File: _includes/header.html [hunk 2]
```diff
@@ -30,6 +30,9 @@
                     <li>
                         <a class="page-scroll" href="http://github.com/screwdriver-cd">GitHub</a>
                     </li>
+                    <li>
```
Comment (stjohnjohnson) [inline _includes/header.html]: I wouldn't put this in the header.

File: _config.yml [hunk 3]
```diff
@@ -30,3 +30,27 @@ exclude:
...
+- name: Pete Peterson
+  pic: petey
+  position: Lead Front-end Architect
+  social:
+    - title: github
+      url: https://github.com/petey
+
+- name: St.John Johnson
+  pic: stjohn
+  position: Lead Core Developer
```
Comment (stjohnjohnson) [inline _config.yml]: Lol sure.  I guess it's better than `Product Owner`.

File: _includes/team.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<!-- Team Section -->
+    <section id="team" class="bg-light-gray">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 text-center">
+                    <h2 class="section-heading">Our Amazing Team</h2>
+                    <h3 class="section-subheading text-muted">Our founders have over 10 years of combined experience building and serving Continuous Delivery products.</h3>
```
Comment (FenrirUnbound) [inline _includes/team.html]: > 10 years of combined experience

🤔  Seems like a small number

File: _includes/team.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+                            {% endfor %}
+
+                        </ul>
+                    </div>
+                </div>
+                {% endfor %}
+            </div>
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+                    <p class="large text-muted">Our experiences have inspired us to set out to build an intuitive, sophisticated tool to boost developer productivity. The Screwdriver team is based in Sunnyvale, CA, and consists of Dao Lam, Darren Matsumoto, Jeremiah Wuenschel, Min Zhang, Noah Katzman, Peter Peterson, St. John Johnson, and Tiffany Kyi.</p>
```
Comment (FenrirUnbound) [inline _includes/team.html]: > Noah Katzman

Yay! 🎉

File: _includes/team.html (hunk 4 above)
Comment (tkyi) [inline _includes/team.html]: I totally made up that number. Anyone have a better one?

File: _config.yml (hunk 3 above)
Comment (tkyi) [inline _config.yml]: Not sure what the titles should be

File: _includes/team.html (hunk 4 above)
Comment (FenrirUnbound) [inline _includes/team.html]: I feel like I'm stuck on the wording; I think it's the "combined experience" part. 

It's small, and I wouldn't consider this a blocking review comment. At the moment I have no better alternative.

Comment (tkyi) [inline _includes/team.html]: Honestly, I'm not sure what constitutes "founders"

Comment (petey) [inline _includes/team.html]: I think stjohn and I each have at least 4 years. For the purpose of this number, I'd include everyone on the list of team members as a "founder".

File: _includes/team.html (hunk 5 above)
Comment (petey) [inline _includes/team.html]: `intuitive, yet sophisticated`

Comment (stjohnjohnson) [inline _includes/team.html]: Do you want to link to your GH accounts?

File: _includes/team.html (hunk 4 above)
Comment (stjohnjohnson) [inline _includes/team.html]: I bet @jer has at least 4 years experience.  I bet you we could just say "over a dozen" :P

File: _includes/team.html (hunk 5 above)
Comment (stjohnjohnson) [inline _includes/team.html]: You could also do the same kind of photo setup, but have everyone at like 50% size so it all fits on the screen.

File: _config.yml [hunk 6]
```diff
@@ -30,3 +30,47 @@ exclude:
...
+  github: https://github.com/nkatzman
+
+- name: Tiffany Kyi
+  pic: tkyi
+  position: Turtle Addict
+  github: https://github.com/tkyi
+
+- name: Dao Lam
+  pic: d2lam
+  position: D2Bully
```
Comment (d2lam) [inline _config.yml]: D2

Comment (tkyi) [inline _config.yml]: :d2what:

File: _config.yml [hunk 7]
```diff
@@ -30,3 +30,47 @@ exclude:
...
+  github: https://github.com/d2lam
+
+- name: Darren Matsumoto
+  pic: FenrirUnbound
+  position: Not a Bounty Hunter
+  github: https://github.com/FenrirUnbound
```
Comment (FenrirUnbound) [inline _config.yml]: Twitter?

https://twitter.com/FenrirUnbound

Comment (FenrirUnbound) [inline _config.yml]: https://twitter.com/FenrirUnbound

File: _config.yml (hunk 6 above)
Comment (d2lam) [inline _config.yml]: :d2please:

File: _config.yml [hunk 8]
```diff
@@ -30,3 +30,47 @@ exclude:
...
+  github: https://github.com/stjohnjohnson
+
+- name: Peter Peterson
+  pic: petey
+  position: Javascript Jedi
+  github: https://github.com/petey
+
+- name: Jeremiah Wuenschel
+  pic: jer
+  position: Not a Jeremiahson
```
Comment (stjohnjohnson) [inline _config.yml]: @jer what do you think?

File: _config.yml (hunk 6 above)
Comment (FenrirUnbound) [inline _config.yml]: :d2awesome:

File: _includes/team.html [hunk 9]
```diff
@@ -0,0 +1,64 @@
+<!-- Team Section -->
+    <section id="team" class="bg-light-gray">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 text-center">
+                    <h2 class="section-heading">Our Amazing Team</h2>
+                    <h3 class="section-subheading text-muted">Our founders have over a dozen years of combined experience building and serving Continuous Delivery products.</h3>
```
Comment (stjohnjohnson) [inline _includes/team.html]: I would remove combined.  It flows better.

File: _includes/team.html [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+            <div class="row">
+                {% for member in site.founders %}
+                <div class="col-sm-4">
+                    <div class="team-member">
+                        <a href="{{ member.github }}">
+                            <img src="img/team/{{ member.pic }}.jpg" class="img-responsive img-circle" alt="">
+                        </a>
+                        <h4>{{ member.name }}</h4>
+                        <p class="text-muted">{{ member.position }}</p>
+                        <!-- <ul class="list-inline social-buttons">
```
Comment (stjohnjohnson) [inline _includes/team.html]: Remove commented code

File: _includes/team.html (hunk 9 above)
Comment (tkyi) [inline _includes/team.html]: I feel like there should be _some_ words there. I just don't know what to put. I'm open to suggestions

File: _config.yml (hunk 7 above)
Comment (tkyi) [inline _config.yml]: sticking with Github links to be consistent 😛

Comment (FenrirUnbound) [inline _config.yml]: 👍

File: _config.yml (hunk 8 above)
Comment (jer) [inline _config.yml]: The same title could be applied to anyone on the team. Not unique enough.

Comment (d2lam) [inline _config.yml]: Lieben's dad

File: _config.yml [hunk 11]
```diff
@@ -30,3 +30,47 @@ exclude:
...
+  github: https://github.com/jer
+
+people:
+- name: Noah Katzman
+  pic: nkatzman
+  position: Hates Cats
```
Comment (nkatzman) [inline _config.yml]: It's good. I'm not a big fan of cats meow

File: _config.yml [hunk 12]
```diff
@@ -30,3 +30,47 @@ exclude:
...
+  github: https://github.com/tkyi
+
+- name: Dao Lam
+  pic: d2lam
+  position: D2Bully
+  github: https://github.com/d2lam
+
+- name: Darren Matsumoto
+  pic: FenrirUnbound
+  position: Not a Bounty Hunter
```
Comment (tkyi) [inline _config.yml]: updated

**Pull request merged**